src/Tests/CrossTargeting.targets

```diff
@@ -0,0 +1,22 @@
+<Project>
+  <!--
+    Arcade uses target batching to run each test assembly TFM for a single test project.
+    This causes subsequent items in the batch to not be run if a prior items fail. For example,
+    if a test project has the 3.1, 5.0, and 6.0 TFMs and its tests are run in that order and
+    the 3.1 TFM has test failures, the 5.0 and 6.0 TFMs will not be run. Override Arcade's
+    RunTests target to use MSBuild to call the Test target for each TFM.
```

Comment:
Does this target change frequently in arcade itself? My concern here is that if Arcade changes this target, we will have difficult to diagnose test run issues.

Reply:
No, it hasn't been changed in years. It will also be very clear that it changed when it does because all legs of the build will fail when it cannot find the RunTests target that we expect in our redefinition of the Test target.